### PR TITLE
To adapt the change in qa_testset_automation, change run file to run.openqa

### PR DESCRIPTION
**Pull request by SeroSun**
This modification is to remove the add repo behavior during run qa_testset_automation to avoid dependency. 
After this PR get merged please help us to add a parameter in our kernel_regression and userspace_regression cases "QA_WEB_REPO=http://dist.suse.de/install/SLP/SLE-12-Module-Web-Scripting-LATEST/%ARCH%/CD1/"

### CODE REVIEW

**Comment by coolo** [comment]
if this is only supposed to be used in some tests, you need to run zypper ar only when the variable has content

**Comment by okurz** [comment]
Please see https://github.com/os-autoinst/os-autoinst/#rules-for-commits especially about the commit message

**Comment by SeroSun** [comment]
@coolo Thank you. I got it, I'll modify zypper ar in a if check.
@okurz It's realy weird, because I already use tidy checked no issue before I commit it. I ln the tidy in the same forder as qa_run.pm, then run ./tidy --check qa_run.pm and no diff show up.

**Comment by SeroSun** [comment]
@okurz I got the reason, because I just link a tify file in this forder not the link the whole tools forder, thanks

**Comment by okurz** [comment]
good to know, thanks. now the commits should be squashed into one. see "git rebase --interactive". Please describe the goal or benefit of your commit. See http://chris.beams.io/posts/git-commit/

**Comment by SeroSun** [comment]
@okurz Thanks, modification has done.

File: tests/qa_automation/qa_run.pm [hunk 1]
```diff
@@ -64,10 +64,17 @@ sub prepare_repos {
...
     my $qa_head_repo = get_var('QA_HEAD_REPO', '');
+    my $qa_web_repo  = get_var('QA_WEB_REPO',  '');
     unless ($qa_head_repo) {
         die "No QA_HEAD_REPO specified!";
     }
     assert_script_run "zypper --no-gpg-check -n ar -f '$qa_head_repo' qa-ibs";
+    if ($qa_web_repo) {
+        assert_script_run "zypper --no-gpg-check -n ar -f '$qa_web_repo' qa-web";
+    }
+    unless ($qa_web_repo) {
```
Comment (okurz) [inline tests/qa_automation/qa_run.pm]: that should be an "else".
```perl
if (…) {
…
}
else {
…
}
```

**Comment by SeroSun** [comment]
Good point, I guess it's OK now @okurz

**Comment by okurz** [comment]
LGTM

**Pull request merged**